src/st/compatibility/class-wpml-page-builders-update-translation.php

```diff
@@ -56,6 +56,5 @@ protected function get_translation( WPML_PB_String $string ) {
 		return $string;
 	}
 
-	abstract protected function update_strings_in_modules( array &$data_array );
```

Comment:
@adrianosferreira you've removed this method which is called in this very class as line #37.

Reply:
@andreasciamanna this is an abstract class, so each lib has its own update translation class. This class is covered on each lib, so if it would fail it wouldn't be here, but in the lib (e.g Cornerstone). Look that it is well implemented there and that's why it didn't fail:

https://github.com/OnTheGoSystems/wpml-page-builders-cornerstone/blob/wpmlcore-5811/src/class-wpml-cornerstone-update-translation.php

Of course, I shouldn't remove this, it was done by mistake. This method should be in the abstract class simply to work like an interface, so child classes know it should implement that. Thanks for pointing it out.